org.hl7.fhir.r4/src/main/java/org/hl7/fhir/r4/utils/client/FHIRToolingClient.java

```diff
@@ -169,9 +173,18 @@ public void setMaximumRecordCount(int maxResultSetSize) {
 	}
 	
 	public TerminologyCapabilities getTerminologyCapabilities() {
-    return (TerminologyCapabilities) utils.issueGetResourceRequest(resourceAddress.resolveMetadataTxCaps(), getPreferredResourceFormat(), "TerminologyCapabilities", TIMEOUT_NORMAL).getReference();
+  	  try {
+		return (TerminologyCapabilities) utils.issueGetResourceRequest(resourceAddress.resolveMetadataTxCaps(), getPreferredResourceFormat(), "TerminologyCapabilities", TIMEOUT_NORMAL).getReference();
+	  } catch(Exception e) {
+		  // THIS IS A WORKAROUND FOR THE TERMINOLOGY SERVER RESPONDING WITH A 404 TO /metadata
```

Comment:
If the call is returning a 404, we need to deal with the 404 and not put code in the `catch` block to execute. This is not good practice.